runtime/compiler/optimizer/J9EstimateCodeSize.cpp

```diff
@@ -1712,26 +1718,26 @@ TR_J9EstimateCodeSize::realEstimateCodeSize(TR_CallTarget *calltarget, TR_CallSt
                continue;
                }
 
-            if (_optimisticSize <= sizeThreshold) // for multiple calltargets, is this the desired behaviour?
+            if (_analyzedSize <= allowanceFactor*sizeThreshold) // for multiple calltargets, is this the desired behaviour?
```

Comment:
Similar to the above, looks like we are not adding a new check but making existing one more relaxed?

Reply:
See prior answer https://github.com/eclipse-openj9/openj9/pull/20904#discussion_r1922799320